nb2/api.py

```diff
@@ -1,86 +1,150 @@
-from flask import Blueprint, jsonify, request
+from flask import Blueprint
+from flask_restful import abort, Api, fields, marshal, reqparse, Resource
+from sqlalchemy.exc import IntegrityError
 
-from nb2 import db
-from nb2.errors import conflict, does_not_exist_error, validation_error
 from nb2.models import Person, Quote
-from nb2.validators import Validators
 
-bp = Blueprint('api', __name__)
 
+bp = Blueprint("api", __name__)
+api = Api(bp)
 
-@bp.route('/')
-def hello():
-    return 'Hello'
 
+class PersonResourceBase(Resource):
 
-@bp.route('/people', methods=['GET'])
-def get_all_people():
-    return jsonify([person.serialize() for person in Person.query.all()])
+    def __init__(self, *args, **kwargs):
+        self.fields = {
+            "id": fields.Integer,
+            "slack_user_id": fields.String,
+            "first_name": fields.String,
+            "last_name": fields.String,
+            "quotes": fields.List(fields.String(attribute='content')),
+        }
 
+        self.parser = reqparse.RequestParser(bundle_errors=True)
 
-@bp.route('/people/<slack_user_id>', methods=['GET'])
-def get_person(slack_user_id):
-    person = Person.query.filter_by(slack_user_id=slack_user_id).first()
+        super(PersonResourceBase, self).__init__(*args, **kwargs)
```

Comment:
Based on [this answer](https://stackoverflow.com/a/576183), it seems like we can just do the following here?:

```suggestion
        super().__init__(*args, **kwargs)
```

Reply:
oh nice! I'll change it!